creator-node/src/services/sync/primarySyncFromSecondary.js

```diff
@@ -185,11 +193,19 @@ async function _primarySyncFromSecondary({
         throw e
       }
 
-      const clockInfo = fetchedCNodeUser.clockInfo
-      if (clockInfo.localClockMax <= clockInfo.requestedClockRangeMax) {
+      /**
+       * TODO update this once all nodes are running 0.3.66
+       * If localClockMax < requestedClockRangeMax, we know there is no more data to fetch
+       * If localClockMax > requestedClockRangeMax, we know there is more data to fetch
+       * If localClockMax = requestedClockRangeMax, there are 2 possibilities:
+       *  1. either the node is running behind version 0.3.66, has more data, and is incorrectly reporting it
```

Comment:
I think I see what you're doing here but I'll let Dheeraj confirm since I don't remember enough of the previous bug to confirm that the clock returned would always be equal in only these 2 scenarios

Reply:
okay i see what's going on here. i think this is fine because you changed this to <. wrote out a few cases to sanity check the export condition

fetchedLocalClockMax - 9998, requestedClockRangeMax - 9999, completed = true
fetchedLocalClockMax - 9999, requestedClockRangeMax - 9999, completed = false, exportClockRangeMin = 10000
fetchedLocalClockMax - 10500, requestedClockRangeMax - 9999, completed = false, exportClockRangeMin = 10000

Reply:
exactly this